_posts/2017-01-08-2016-a-year-dedicated-to-python-workshops.md

```diff
@@ -0,0 +1,265 @@
+tag: pythonworkshop
+---
+
+Beautiful 2017 has already started. While everybody is busy with preparing 
+resolutions for their new year I decided to look back and share my journey
+here.
+
+After attending [PyCon India 2015](https://in.pycon.org/2015/) India 2015 I
+took an oath to spread the word of Python. It was looking simple at first
+glance with a guess of getting less invitations. But the estimate went wrong.
+Below is the summary of yearly progress highlight with word of experience.
+
+
+<link rel="stylesheet" type="text/css" href="{{site.url}}/assets/css/nv.d3.min.css">
+
+<style>
+    #chart svg {
+      height: 400px;
+    }
+
+    svg: {
+      display: block;
+    }
+
+    svg text {
+      font: normal 8px Impact;
+    }
+
+    .center {
+      text-align: center
+    }
+
+</style>
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.2/d3.min.js" charset="utf-8"></script>
+
+<script src="{{ site.url }}/assets/js/nv.d3.min.js"></script>
+
+<script>
+
+function GetData() {
+  return  [
+    {
+      "label": "Introduction to Django",
+      "value" : 1
+    },
+    {
+      "label": "My name is Python and I am not terrorist",
+      "value" : 1
+    },
+    {
+      "label": "Beginning with Python",
+      "value" : 2
+    },
+    {
+      "label": "Let's learn Python",
+      "value" : 5
+    },
+    {
+      "label": "Token based authentication system using JWT",
+      "value" : 4
+    },
+  ];
+}
+
+nv.addGraph(function() {
+  var chart = nv.models.pieChart()
+      .x(function(d) { return d.label })
+      .y(function(d) { return d.value })
+      .showLabels(true)
+      .labelType("value");
+
+  d3.select("#chart svg")
+      .datum(GetData())
+      .transition().duration(350)
+      .call(chart);
+
+  return chart;
+});
+
+</script>
+
+<div id="chart">
+  <svg></svg>
+</div>
+
+
+During my last year I conducted 13 workshops at various colleges of
+state Gujarat and Rajasthan. 10 workshops were conducted targeting college
+students and 3 were presented for professionals. Gandhinagar and Ahmedabad
+received 4 number of workshops individually. Did single workshop at Bhuj,
+Ajmer, Vadodara and New delhi. One over Google Hangouts on Air.
+
+
+Facilities I received while conducting workshops
+------------------------------------------------
+
+The colleges which were settled far from my town provided a travelling
+allowance for me. Which I accepted because I felt there is nothing wrong in
+taking such help. Some colleges arranged an afternoon meal for me. I happily
+accepted that because finding another option during short breaks at unknown
+place is quite time consuming. There were few colleges which gave cash covers 
+as a good will. That I never decided to accept in my lifespan. The reason I
+don’t accept money from any educational institute is they sometimes charge
+students for raising that money. I don’t feel good to take money from the
+pocket of any student. Still some college claimed that the money belongs to
```

Comment:
don’t accept money from any educational institute is they sometimes charge
 students for raising that money. I don’t feel good to take money from the
 pocket of any student.    --->  don't accept money from any educational institute because it sometimes comes from student's pocket.